Jenkinsfile

```diff
@@ -389,17 +388,16 @@ def buildWindows(nodeVersion, arch) {
 
 def inAndroidContainer(workerFunction) {
   return {
-    myNode('android') {
+    myNode('docker-cph-03') {
```

Comment:
Why can we only use `docker-cph-03`?

Reply:
that's the current code Java uses. I can not say for sure

Reply:
If we don't know why we're changing this, I would rather that we leave it as is until we know for sure. Alternatively reach our to Java to understand why they feel the need to do this.

Reply:
we know exactly why we are changing this. This is the only node which is able to run Android emulator and Android tests currently on our Jenkins infrastructure. Leaving it as it is is not an option